app/views/layouts/application.html.erb

```diff
@@ -1,8 +1,9 @@
 <!DOCTYPE html>
```

Comment:
Please update the README file as was asked in the previous review. A professional README should have a project title, description of its features, and a set of instructions to run the app locally.